template-processors/helm-oc/Dockerfile.in

```diff
@@ -0,0 +1,11 @@
+FROM @REPOSITORY@/eunomia-helm:@IMAGE_TAG@
+
+ENV OC_VERSION="3.11.165" \
+    kubectl=oc
```

Comment:
Shouldn't it be an alias instead of env variable?

Reply:
It's an env variable that was added already by somebody else for the base image. We can change that, but it'll involve a more intrusive PR. I think it might be clearer for now that this uses `oc` instead of `kubectl`